MAINTAINERS.md

```diff
@@ -96,7 +96,7 @@ We currently use ZenHub only with our public and opensource repositories.
 ### Setting up ZenHub
 
 ZenHub can be easily added as an [extension]((https://www.zenhub.com/extension)) to chrome and firefox which can be downloaded for free from the ZenHub website. 
-Alternatively, we can use a the ZenHub [webapp link](https://app.zenhub.com/workspaces/engineering-effectiveness-614cf4272a385f0015d2b48f/board?repos=357723952,406037663) to view the board.
+Alternatively, we can use the ZenHub [webapp link](https://app.zenhub.com/workspaces/engineering-effectiveness-614cf4272a385f0015d2b48f/board?repos=357723952,406037663) to view the board.
```

Comment:
Zenhub is not being used anymore by this project. Can you please remove line 87 to 99?
I believe we have an image in root of the repo related to zenhub that can be removed too.

Reply:
I have removed all the irrelevant details related to Zenhub (lines 87 to 173).